l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1012,6 +1012,11 @@ def set_vendor_bill_data(self, FatturaBody, invoice):
                 'reference':
                     FatturaBody.DatiGenerali.DatiGeneraliDocumento.Numero,
             })
+        if not invoice.supplier_invoice_number:
```

Comment:
Puoi aggiungere il controllo di questo campo in un test come già succede per `reference`?

Reply:
fatto grazie.

Reply:
👍🏼